reasoner/udf/src/main/java/com/antgroup/openspg/reasoner/udf/UdfMng.java

```diff
@@ -19,10 +19,11 @@
 import com.antgroup.openspg.reasoner.udf.model.UdafMeta;
 import com.antgroup.openspg.reasoner.udf.model.UdfParameterTypeHint;
 import com.antgroup.openspg.reasoner.udf.model.UdtfMeta;
+import java.io.Serializable;
 import java.util.List;
 import java.util.Map;
 
-public interface UdfMng {
+public interface UdfMng extends Serializable {
```

Comment:
This class is not allowed to be serialized as it contains a Java Method Object internally.
you can call UdfMngFactory.getUdfMng to retrieve an instance

Reply:
fixed

> This class is not allowed to be serialized as it contains a Java Method Object internally. you can call UdfMngFactory.getUdfMng to retrieve an instance